updatecli/updatecli.d/Mogztter-kroki.yml

```
@@ -0,0 +1,59 @@
+---
+title: "Bump Asciidoctor-Kroki version"
+sources:
+  getAsciidoctorKrokiVersion:
+    kind: githubRelease
+    name: "Get the latest Asciidoctor-Kroki version"
+    spec:
+      owner: "Mogztter"
+      repository: "asciidoctor-kroki"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: latest
+    transformers:
+      - trimPrefix: "v"
```

```suggestion
      versionFilter:
        kind: regex
        pattern: 'ruby-*'
    transformers:
      - trimPrefix: "ruby-v"
```
=> the "latest" will catch whataver asciidoctor-version, which happen to be the `javascript-0.14.0` while writing this comment. By specifying a pattern, you ensure that you lock on the Ruby version: did I understand correctly you intent?

---

you are correct, we need the ruby version not the javascript